src/components/common/previewTable/previewTable.tsx

```
@@ -0,0 +1,51 @@
+} from "@chakra-ui/react";
+
+interface Column {
+  key: string;
+  title: string;
+  dataKey?: string;
+  render?: (value: any, row: any) => React.ReactNode;
+}
+
+interface TableProps {
+  columns: Column[];
+  data: any[];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)